### PR TITLE
Adding isbot and exposing session token for bots to use

### PR DESCRIPTION
**Pull request by BrandonMurnane**

### CODE REVIEW

File: api/v1/controllers/bots.js [hunk 1]
```diff
@@ -169,6 +175,7 @@ module.exports = {
   putBots(req, res, next) {
     const resultObj = { reqStartTime: req.timestamp };
     const reqObj = req.swagger.params;
+    const seqObj = {};
```
Comment (kriscfoster) [inline api/v1/controllers/bots.js]: @BrandonMurnane think you should be doing something with this before line 200

File: config/toggles.js [hunk 2]
```diff
@@ -183,6 +183,10 @@ const shortTermToggles = {
   // require helpEmail or helpUrl in POST/PUT/PATCH of aspects and subjects
   requireHelpEmailOrHelpUrl: environmentVariableTrue(
     pe, 'REQUIRE_HELP_EMAIL_OR_HELP_URL'),
+
+  // adds isbot to token and returns token on patches not just posts
```
Comment (kriscfoster) [inline config/toggles.js]: @BrandonMurnane IsBot

**Comment by iamigo** [comment]
What's the status of this PR?

**Comment by BrandonMurnane** [comment]
Ready to merge

On 28 Nov 2018, at 16:50, Ian Goldstein <notifications@github.com> wrote:

What's the status of this PR?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/salesforce/refocus/pull/1040#issuecomment-442519322>,
or mute the thread

**Comment by harshkothari410** [comment]
You need to edit some code in socket connection as well. So that socket can connect using your bot token instead of general token.

**Pull request merged**